src/ts/background/store/index.ts

```diff
@@ -1,16 +1,5 @@
 import { createStore, applyMiddleware } from 'redux'
 import promise from 'redux-promise-middleware'
-import thunkMiddleware from 'redux-thunk'
-import { loadingBarMiddleware } from 'react-redux-loading-bar'
-import reducers, { IAppState } from './all'
+import reducers from './all'
 
-const defaultMiddlewares = [
-  thunkMiddleware,
```

Comment:
So we do not need `thunkMiddleware` anymore?

Reply:
No, we have never used thunk even though we declared. redux-promise-middleware is sufficient now. If we need thunk later, we can always add it back.